powerpc/install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# usage: /new-path/python/environment/
+set -e
+
+VIRTUAL_ENV=$1
```

Comment:
Can we check for the number of arguments here? If none is given, an error should be produced